pylds/tools.py

```diff
@@ -38,56 +39,48 @@ def draw_lagrangian_descriptor(LD, LD_type, GRID_PARAMETERS, LD_PARAMETERS, norm
     -------
         scatter plot of LD function in phase-space 
     """
-    x_min, x_max, Nx = GRID_PARAMETERS[0]
-    y_min, y_max, Ny = GRID_PARAMETERS[1]
-    
-    t_initial, t_final, dt = LD_PARAMETERS[0]
-    p_norm = LD_PARAMETERS[1]
-    
-    tau = t_final - t_initial
-    
-    LD = LD.reshape(Nx, Ny).T # Reshape 1D array
+    x_min, x_max, Nx = grid_parameters[0]
+    y_min, y_max, Ny = grid_parameters[1]
+        
     if norm:
         LD = LD / LD.max()  # Scale LD output
-    ################################### 
+    
     # Plot LDs
-    resolution = 100 # in dpi
-    fig,ax = plt.subplots(1, 1, dpi = resolution)
+    fig,ax = plt.subplots(1, 1, dpi = 100)
     
     points_x = np.linspace(x_min, x_max, Nx)
     points_y = np.linspace(y_min, y_max, Ny)    
-    X, Y = np.meshgrid(points_x, points_y)  # Grid in phase-space
     
-#     scatter = plt.scatter(X, Y, c = LD, cmap = colormap_name)
-    contour = plt.contourf(X,Y,LD,cmap=colormap_name,levels=200)
-    ###################################
+    contour = plt.contourf(points_x,points_y,LD,cmap=colormap_name,levels=200)
```

Comment:
We need to have an agreement on this guys, @vkrajnak @VikJGG 

I can see that `plt.contourf` makes the graphical output look smoother and fixes the distortion in the boundaries of the figure frame previously seen using `scatter`, e.g., Hamilton Saddle, see below

BEFORE with `scatter`
![Screenshot 2020-06-11 at 14 47 48](https://user-images.githubusercontent.com/17453074/84393249-47e7d600-abf3-11ea-8201-e201410158ee.png)

AFTER with `contourf`

![Screenshot 2020-06-11 at 14 51 42](https://user-images.githubusercontent.com/17453074/84393278-4e764d80-abf3-11ea-974a-bda92b46ae8f.png)

Other options we discussed were `pycolor` and `pcolormesh`. According to some posts I've read, `pcolormesh` seem to be faster than `pcolor` in terms of rendering. Although `contourf` gives a better output, the only thing that I see as a disadvantage is that the number of contour lines has to be specified, here `n=200`. In terms of rendering speed, I have no idea how it compares with other options.

Reply:
Scatter draws dots, hence the distortion is due to overlapping dots. Using scatter for an entire surface is definitely not recommended and I introduced contourf solely because of that. As long as pycolor and pcolormesh draw surfaces like contourf, anything will do. I do not think having to specify the number of countour levels is an issue.